docs/source/tts/models.rst

```diff
@@ -104,6 +104,18 @@ Speech-to-text alignment is a critical component of neural TTS models. Autoregre
         :scale: 25%
 
 
+End2End Models
+--------
+
+VITS
+~~~~~~~~~~~~~~~
+VITS is an end-to-end speech synthesis model, which generates raw waveform audios from grapheme/phoneme input. It uses Variational Autoencoder to combine GlowTTS-like spectrogram generator with HiFiGAN vocoder model. Also is has separate flow-based duration predictor, which samples alignments from noise with conditioning on text.  Please refer to :cite:`tts-models-kim2021conditional` for details.
```

Comment:
Nitpicks:
s/HiFiGAN/HiFi-GAN
s/Also is has separate/Also, it has a separate

Reply:
Also, might be worth mentioning that it is `@experimental` right now, unless you plan to remove that tag soon.